docs/variants/dell_optiplex/recovery.md

```diff
@@ -1,5 +1,205 @@
+|Dell    | OptiPlex 7010 DT |
+|Dell    | OptiPlex 9010 SFF |
+
+</center>
+
+## Step 1: Prepare hardware
+
+1. To prepare hardware please follow
+[hardware preparation](../initial-deployment/#hardware-preparation) procedure.
+2. Follow instruction in the video to remove heatsink:
+  <center>
+  <iframe width="640" height="480"
+    src="http://www.youtube.com/embed/TiUSTo-XwPo">
+  </iframe>
+  </center>
+## Step 2: Find SPI_1, SPI_2/SPI_3
+
+<center>
+![](../../images/dell_optiplex_9010_spi_pin1.jpg)
+</center>
+
+## Step 3: Connect SOIC-8 Pomona clip between RTE and target
+
+Connect SOIC-8 Pomona according to [MX25L3206E datasheet](https://www.macronix.com/Lists/Datasheet/Attachments/8616/MX25L3206E,%203V,%2032Mb,%20v1.5.pdf).
+
+<center>
+![](../../images/mx25l3206e_pinout.png)
+</center>
+
+<center>
+ ![Pomona SOIC clip](../../images/pomona_clip.jpg)
+</center>
+
+<center>
+
+ | RTE J7                                 | Pomona SOIC clip  |
+ |:--------------------------------------:|:-----------------:|
+ | CS                                     | pin 1 (upside)    |
+ | MISO                                   | pin 2 (upside)    |
+ | GND                                    | pin 4 (upside)    |
+ | VCC                                    | pin 5 (downside)  |
+ | SCLK                                   | pin 7 (downside)  |
+ | MOSI                                   | pin 8 (downside)  |
+
+</center>
+
+Numbers 1-4 have to be on one side and numbers 5-8 have to be on the other side
+of the clip.
+
+<center>
+![Clip up](../../images/clip_upside.jpg)
+</center>
+
+<center>
+![Clip down](../../images/clip_downside.jpg)
+</center>
+
+Clip on the `SPI_1` chip. Match pin 1 (`CS`) of the Pomona clip with the first
+pin of `SPI_1` chip, marked with a small dot engraved on the chip.
+
+![Clip connected](../../images/clip_connected.jpg)
+
+## Step 4: Connect RTE
+
+<center>
+![](../../images/rte-boot.jpg)
+</center>
+
+1. Connect J2 Orange Pi Zero system debug output
+2. Power the board and confirm it boots
+3. **Please note** typical convention of USB-UART converter colors is as follows
+    - black - GND
+    - red - +5V
+    - green - TX
+    - white - RX
+4. Connect terminal to RTE and read OS version:
+
+    ```shell
+    sudo minicom -b 115200 -D /dev/ttyUSB0 -o -C /tmp/minicom.cap
+    ```
+
+    - `-b 115200` sets baudrate
+    - `-D /dev/ttyUSB0` points to USB-UART converter device, it can be
+      different if you already have some devices connected or you use different
+      operating system
+    - `-o` skip initialization
+    - `-C /tmp/minicom.cap` capture serial terminal output, if you will have
+      problems with exercises please post this file
+5. Login using following credentials:
+
+    ```shell
+    login: root
+    password: meta-rte
+    ```
+
+## Step 5: Prepare recovery binary
+
+* Get v0.2-rc3 binary
+
+```bash
+wget https://cloud.3mdeb.com/index.php/s/8WNEHEFcBGFRK23/download -O dasharo_dell_optiplex_9010_v0.2-rc3.rom
```

Comment:
is it a really good idea to use this binary?

Reply:
Do we have better idea for now?

Reply:
I do not necessarily want people around the world to have my MAC address...

Reply:
@miczyg1 ok, so this is not just BIOS region?

Reply:
No, it is not just BIOS. If it was BIOS region only, I wouldn't object to using the binary.

Reply:
If we are using it for BIOS recovery only, I may strip it from non-BIOS regions and put a copy of the current file with different name.

Reply:
I'm ok with testing it during OptiPlex Tuesday

Reply:
Maybe we should add event to our calendar? https://3mdeb.com/events/
I might join and discuss as well, especially if there are some deployment/automation etc considerations

Reply:
Done.